src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -215,69 +282,68 @@ save_dh_params(DH *p, const char *filename) {
   return;
 }
 
-static DH *dh1024_tmp = NULL, *dh2048_tmp = NULL;
-static const char *dh1024_file = NULL, *dh2048_file = NULL;
 static int
 generate_dh_params(eventer_t e, int mask, void *cl, struct timeval *now) {
   (void)e;
   (void)now;
-  int bits = (int)(intptr_t)cl;
+  struct stat sb;
+  char errstr[120];
```

Comment:
nit - magic 120 length like what Phil recently used although he set it at 65535

Reply:
120 is specified by the OpenSSL docs and mentioned elsewhere in the code. I'll try to make this more obvious,.

Reply:
that would be good, give it a #define